root/frontend/src/styles/PreviewCard.scss

```diff
@@ -0,0 +1,63 @@
+.previewCard {
+	width: 200px;
+	background: #4e525b;
+	max-height: 250px;
+	border-radius: 15px;
+}
+
+.previewCardHeader {
+	display: flex;
+	align-items: center;
+	height: 50px;
+	width: 100%;
+}
+
+.profileImage {
+	border-radius: 100%;
+	width: 30px;
+	height: 30px;
+	margin: 0px 10px 0px 10px;
+}
+.previewCardTitle {
+	color: white;
+	height: 33px;
+	white-space: nowrap;
+	overflow: hidden;
+	text-overflow: ellipsis;
+	max-width: 150px;
+}
+.list {
+	list-style-type: none;
+	margin-top: 0px;
+    :nth-child(n+7) {
+        display: none;
+    }
+}
+.diamond {
```

Comment:
This class is never used. I would recommend adding this to the diamond icons for the list items and poossibly adding a margin to space out the bullet point and text.